tests/Feature/API/WpOrg/ApiAuthenticationTest.php

```diff
@@ -1,11 +1,13 @@
 <?php
 
-it('should return 401 if the API authentication is enable', function () {
+it('should return 200 when no auth token is present', function () {
     $response = $this->getjson('/plugins/info/1.1?action=query_plugins');
+    $response->assertStatus(200);
+});
+
+it('should return 401 when an invalid auth token is present', function () {
+    $response = $this->getjson('/plugins/info/1.1?action=query_plugins', ['Authorization' => 'Bearer invalid-token']);
     $response->assertStatus(401);
-})->skip(fn() => !config('app.aspirecloud.api_authentication_enable'), 'API authentication is disabled');
+});
 
-it('should return 200 if the API authentication is disable', function () {
-    $response = $this->getjson('/plugins/info/1.1?action=query_plugins');
-    $response->assertStatus(200);
-})->skip(fn() => config('app.aspirecloud.api_authentication_enable'), 'API authentication is enabled');
+// TODO: write tests for real and and bad auth tokens
```

Comment:
```suggestion
// TODO: write tests for real and bad auth tokens
```
Double "and". Possibly jittery from too much coffee :wink:

Reply:
☕️☕️☕️ + 🌲🌲 = 😳

Anyway I already tested a bad token so I just need to generate and use a real one, and hopefully I'll have that knocked out before I merge this (it's already getting deployed to .io before then)